collaborative/IOU/heat_index.ipynb

```diff
@@ -0,0 +1,557 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d80a22d3-263d-4d47-8031-a1d9515f92eb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# select data\n",
+    "selections = ck.Select()\n",
+    "\n",
+    "selections.data_type = 'Gridded'\n",
+    "selections.timescale = 'hourly'\n",
+    "selections.variable_type='Derived Index'\n",
+    "selections.variable='NOAA Heat Index'\n",
+    "selections.resolution = '9 km'\n",
+    "selections.time_slice = (1981, 2010)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51db9824-fd69-4b6b-afa8-71033458a953",
+   "metadata": {},
+   "source": [
+    "Will use Fresno as an example first. For example, the following stations are within PG&E area: Arcata, Red Bluff, Stockton, San Jose, Fresno, San Luis Obisbpo, Bakersfield."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1b78440b-7099-4416-9317-60e89ee7c1bd",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from climakitae.core.paths import stations_csv_path\n",
+    "wx_stns = read_csv_file(stations_csv_path, index_col=[0])\n",
+    "wx_stns.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9972811a-b418-4841-991e-849e502ac612",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "station_name = 'Fresno Yosemite International Airport (KFAT)'\n",
+    "one_stn = wx_stns.loc[wx_stns['station'] == station_name]\n",
+    "\n",
+    "stn_lat = one_stn.LAT_Y.values[0]\n",
+    "stn_lon = one_stn.LON_X.values[0]\n",
+    "print(stn_lat, stn_lon)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c041cb58-5951-454e-aeae-f6280331c477",
+   "metadata": {},
+   "source": [
+    "If you would like to provide your own latitude and longitude coordinates, you can also customize the cell below and pass your own values. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d3258596-3c08-476d-b9ad-ab69a604c747",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# stn_lat = YOUR_LAT_HERE\n",
+    "# stn_lon = YOUR_LON_HERE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8b0e172-6423-4551-b207-f445ba24e16b",
+   "metadata": {},
+   "source": [
+    "Next, we'll use the latitude and longitude values to retrieve the historical data at that gridcell. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "09fba876-5f97-4262-b80e-a5ceb226e8db",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "selections.latitude = (stn_lat - 0.05, stn_lat + 0.05)\n",
+    "selections.longitude = (stn_lon - 0.05, stn_lon + 0.05)\n",
+    "\n",
+    "# because we're retrieving a single grid cell, we also take the area average\n",
```

Comment:
could be multiple grid cells (e.g. Burbank-Glendale-Pasadena Airport (KBUR)), which is no problem. The values would just be weighted area averages.

Reply:
get_closest_gridcell makes sense! I noticed the `selections.cached_area` is set to PG&E, so get_closest_gridcell wouldn't return the expected data if the input lat/lon is outside of the PG&E zone. If you think the stakeholders might do that for some reason, we'd need a workaround. 

This line below gives the PG&E stations you listed. Feel free to take or leave.
`wx_stns[wx_stns.station.str.contains('|'.join(['Arcata', 'Red Bluff', 'Stockton', 'San Jose', 'Fresno', 'San Luis Obisbpo', 'Bakersfield']))]`